runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -2916,6 +2916,69 @@ public void testActiveThreadMetric() throws Exception {
     executor.shutdown();
   }
 
+  @Test
+  public void testOverrideMaximumThreadCount() throws Exception {
+    int maxThreads = 5;
+    int threadExpirationSec = 60;
```

Comment:
this doesn't matter right? If so set it higher so it isn't possible that it triggers.

Reply:
Cool!